cmake/ExtraRepositoriesList.cmake

```diff
@@ -1,10 +1,16 @@
 SET_DEFAULT_AND_FROM_ENV(Futility_URL_REPO_BASE
   $ENV{USER}@ners-arc-05.engin.umich.edu:/git-root/MPACT/)
-  
+
 #
 # Define the extra repos
 #
-TRIBITS_PROJECT_DEFINE_EXTRA_REPOSITORIES(
-  Trilinos           ""  GIT  ${Futility_URL_REPO_BASE}Trilinos            PRE         Continuous
-  TeuchosWrappersExt ""  GIT  ${Futility_URL_REPO_BASE}TeuchosWrappersExt  PRE         Continuous
-)
+IF(NOT ${PROJECT_NAME}_TRILINOS_TPL)
+  TRIBITS_PROJECT_DEFINE_EXTRA_REPOSITORIES(
+    Trilinos           ""  GIT  ${Futility_URL_REPO_BASE}Trilinos            PRE         Continuous
+    TeuchosWrappersExt ""  GIT  ${Futility_URL_REPO_BASE}TeuchosWrappersExt  PRE         Continuous
+  )
+ELSE()
+  TRIBITS_PROJECT_DEFINE_EXTRA_REPOSITORIES(
```

Comment:
Why is FakeRepo needed?

Reply:
So, I need to follow up with @bartlettroscoe about this.  If the file ExtraRepositoriesList is included we must call `TRIBITS_PROJECT_DEFINE_EXTRA_REPOSITORIES` but that must have at least one repo listed.  I realize that this isn't idea but the other option is to prevent Futility to include the ExtraRepositoriesList file which I figured would add more confusion if we ever add a new extra repo which we don't prebuild.